src/components/layout/Footer.js

```diff
@@ -1,37 +1,44 @@
 import React from 'react';
+import { Link } from 'gatsby';
 
 const Footer = () => (
   <footer className="container mx-auto py-16 px-3 mt-48 mb-8 text-gray-800">
     <div className="flex -mx-3">
-      <div className="flex-1 px-3">
+      <section className="flex-1 px-3">
         <h2 className="text-lg font-semibold">About Us</h2>
-        <p className="mt-5">Ridiculus mus mauris vitae ultricies leo integer malesuada nunc.</p>
-      </div>
-      <div className="flex-1 px-3">
+        <p className="mt-5">
+          How freelancers connect with peers to share gigs and stay up-to-date.
+        </p>
+      </section>
+      <section className="flex-1 px-3">
         <h2 className="text-lg font-semibold">Important Links</h2>
-        <ul className="mt-4 leading-loose list-none">
-          <li>
-            <a href="../../code-of-conduct/">Code of Conduct</a>
-          </li>
-          <li>
-            <a href="../../privacy-policy/">Privacy Policy</a>
-          </li>
-        </ul>
-      </div>
-      <div className="flex-1 px-3">
+        <nav>
```

Comment:
Uses `nav` for sections that hold navigation links. See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav